test/index.spec.js

```diff
@@ -12,42 +12,165 @@ log.getLogger('disciplLawReg').setLevel('warn')
 
 describe('discipl-law-reg', () => {
   describe('The discipl-law-reg library', () => {
-    it('correctly parses a single fact', async () => {
-      let parsedFact3 = lawReg.evaluateFactFunction('[fact1]')
+    it('correctly parses and solves a single fact', async () => {
+      let parsedFact = lawReg.evaluateFactFunction('[fact1]')
 
-      expect(parsedFact3).to.deep.equal(
-        'fact1'
+      log.getLogger('disciplLawReg').setLevel('debug')
```

Comment:
Let's remove this to not clutter the standard test output (also goes for the other 'setLevel' statements in the other tests)